### PR TITLE
Chore: update documentation

### LINKED ISSUE #561: Email `info@dataspace-connector.de` Does Not Exist
**Describe the bug**
main:latest
The email reference `info@dataspace-connector.de` doesn't exist and is used on the pages:
* https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/docs/pages/deployment/update.md
* https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/docs/pages/communication/v6/ecosystem/broker.md
* https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/docs/pages/communication/v6/ecosystem/identityProvider.md
* https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/docs/pages/features.md
* https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/docs/pages/communication.md

**To Reproduce**
1. Send mail to 'info@dataspace-connector.de'
2. See error: One should get an error that the recipient address is rejected due to unknown user.

**Expected behavior**
Existing and active email account

**Desktop (please complete the following information):**
OS: Ubuntu 20.04 LTS
Browser: chrome
Version (DSC): 107.0.5304.110

### PR DESCRIPTION
**Pull request by omarsilva1**
I changed the email address to the currently used support address. I've also added additional information regarding deployment at sovity. 

Closes #561

### CODE REVIEW

File: docs/pages/communication/v6/ecosystem/broker.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The IDS Metadata Broker is available at
 [https://broker.ids.isst.fraunhofer.de](https://broker.ids.isst.fraunhofer.de). It expects IDS
 multipart messages at [https://broker.ids.isst.fraunhofer.de/infrastructure](https://broker.ids.isst.fraunhofer.de/infrastructure).
 The GUI can be accessed at [https://broker.ids.isst.fraunhofer.de/browse](https://broker.ids.isst.fraunhofer.de/browse).
-To get your IP address unblocked, please contact [us](mailto:info@dataspace-connector.de).
+To get your IP address unblocked, please contact [us](mailto:support@sovity.de).
```
Comment (SebastianOpriel) [inline docs/pages/communication/v6/ecosystem/broker.md:23]: Please revise text above

File: docs/pages/communication/v6/ecosystem/identityProvider.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ A mailing list dealing with DAPS certificates and questions regarding the key ma
 The Participant Information System (ParIS) is available at [https://paris.ids.isst.fraunhofer.de](https://paris.ids.isst.fraunhofer.de).
 It expects IDS multipart messages at [https://paris.ids.isst.fraunhofer.de/infrastructure](https://paris.ids.isst.fraunhofer.de/infrastructure).
 The GUI can be accessed at [https://paris.ids.isst.fraunhofer.de/browse](https://paris.ids.isst.fraunhofer.de/browse).
-To get your IP address unblocked, please contact [us](mailto:info@dataspace-connector.de).
+To get your IP address unblocked, please contact [us](mailto:support@sovity.de).
```
Comment (SebastianOpriel) [inline docs/pages/communication/v6/ecosystem/identityProvider.md:68]: Revise text above

File: docs/pages/deployment/update.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ as the latest release. This will look as follows, e.g.:
   "contact": {
     "organization": "Fraunhofer Institute for Software and Systems Engineering",
     "website": "https://www.dataspace-connector.io/",
-    "email": "info@dataspace-connector.de"
+    "email": "support@sovity.de"
```
Comment (SebastianOpriel) [inline docs/pages/deployment/update.md:37]: Please revise text above

**Review by ghost — APPROVED** [review]

File: docs/pages/deployment/update.md (hunk 3 above)
Comment (omarsilva1) [reply docs/pages/deployment/update.md:37]: Fixed in https://github.com/International-Data-Spaces-Association/DataspaceConnector/pull/587

File: docs/pages/communication/v6/ecosystem/identityProvider.md (hunk 2 above)
Comment (omarsilva1) [reply docs/pages/communication/v6/ecosystem/identityProvider.md:68]: Fixed in https://github.com/International-Data-Spaces-Association/DataspaceConnector/pull/587

File: docs/pages/communication/v6/ecosystem/broker.md (hunk 1 above)
Comment (omarsilva1) [reply docs/pages/communication/v6/ecosystem/broker.md:23]: Fixed in https://github.com/International-Data-Spaces-Association/DataspaceConnector/pull/587

**Pull request merged**